pytest-embedded-idf/tests/test_idf.py

```diff
@@ -4,6 +4,11 @@
 import pytest
 from pytest_embedded_idf.dut import IdfDut
 
+toolchain_required = pytest.mark.skipif(
+    os.getenv('PATH') is None or 'riscv32-esp-elf-gdb/bin' not in os.getenv('PATH'),
```

Comment:
the `'riscv32-esp-elf-gdb/bin'` won't work on windows since they're using a different dir separator. `os.path.join` instead